cmd/recoveretcdcluster/recoveretcdcluster.go

```diff
@@ -22,15 +23,11 @@ func main() {
 		log.Fatal("Usage recoveretcdcluster <blobname>")
 	}
 	blobName := os.Args[1]
-	conf, err := fakerp.NewConfig(log, true)
-	if err != nil {
-		log.Fatal(err)
-	}
 
 	ctx := context.Background()
-	ctx = context.WithValue(ctx, api.ContextKeyClientID, conf.ClientID)
-	ctx = context.WithValue(ctx, api.ContextKeyClientSecret, conf.ClientSecret)
-	ctx = context.WithValue(ctx, api.ContextKeyTenantID, conf.TenantID)
+	ctx = context.WithValue(ctx, api.ContextKeyClientID, os.Getenv("AZURE_CLIENT_ID"))
```

Comment:
I sort of feel this is going in the wrong direction, maybe it should mirror how client.go loads its configuration via envconfig? Is there any hard reason why these are environment variables now?

Reply:
Using something like envconfig can be a follow-up, for start I wanted to remove the unnecessary dependency.